site/en/docs/extensions/support-feedback/submit-a-feature-request/index.md

```diff
@@ -0,0 +1,23 @@
+---
+layout: 'layouts/doc-post.njk'
+title: Submit a feature request
+description: 'TBD'
+date: 2023-10-11
+tags:
+  - extensions
+---
+
+## Submit a feature request
+
+If you identify a feature (especially if it's related to an experimental API) that you believe could improve the extension's platform, file a request as described below.
```

Comment:
I like the idea here but given we don't really define experimental API here, I think it's probably worth dropping:

```suggestion
If you identify a feature that you believe could improve the extension's platform, file a request as described below.
```